### PR TITLE
fix(unit-tests): fixed unit test typos and config

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "server": {
-        "port": 3000
+        "port": 3009
```
Comment (asafMasa) [inline config/default.json:3]: check if this file need to be removed and update the conf.d file

File: services/WorkflowsService.js [hunk 2]
```diff
@@ -47,10 +47,10 @@ exports.workflowsPOST = async function (args, res, next) {
 
 exports.workflowsGET = async function (args, res, next) {
   try {
-    logger.info(`[WorkflowService] workflowsGET - get workflows: ${files}`);
     const files = await DataHandlerFileSystem.getFilesFromRootPath(
```
Comment (asafMasa) [inline services/WorkflowsService.js:50]: message received "debug" level

File: test/services/apiInvoker.spec.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ describe("apiInvoker functionality", function () {
       result.should.to.eql(expectedResult);
     });
 
-    it("Should return undefined by pass incorrect params", function () {
+    it("Should return undefined when recive undefined params", function () {
```
Comment (asafMasa) [inline test/services/apiInvoker.spec.js:50]: receive - receiving

**Review by asafMasa — APPROVED** [review]

File: config/default.json (hunk 1 above)
Comment (CL-SHLOMIKONCHA) [reply config/default.json]: fixed in 56b66c

File: test/services/apiInvoker.spec.js (hunk 3 above)
Comment (CL-SHLOMIKONCHA) [reply test/services/apiInvoker.spec.js]: fixed in 56b66c7

File: services/WorkflowsService.js (hunk 2 above)
Comment (CL-SHLOMIKONCHA) [reply services/WorkflowsService.js:51]: fixed in 56b66c7

**Pull request merged**